app/app.go

```diff
@@ -184,7 +184,6 @@ var (
 		ibctm.AppModuleBasic{},
 		ica.AppModuleBasic{},
 		ibcfee.AppModuleBasic{},
-		consensus.AppModuleBasic{},
```

Comment:
registered twice

Reply:
Is that a problem?

Reply:
https://github.com/palomachain/paloma/blob/f2e6d757e296e89c09e4456286b49bb06a71524c/app/app.go#L181
https://github.com/palomachain/paloma/blob/f2e6d757e296e89c09e4456286b49bb06a71524c/app/app.go#L187
It's not required to add it twice, right?

Reply:
No. It's not. Let's remove one so we keep only one.